### PR TITLE
Post-Edit Profile-Follow & Unfollow

### PR DESCRIPTION
**Pull request by horsemaker**

### CODE REVIEW

**Review by narharikale — APPROVED** [review]
clean code 
well structured
required error handling
liked the services implementation

**Review by MarkVed17 — APPROVED** [review]
Looks clean! 👌
Good work here.

File: src/backend/db/posts.js [hunk 1]
```diff
@@ -1405,8 +1404,8 @@ export const posts = [
     username: "LahaneVedant",
```
Comment (MarkVed17) [inline src/backend/db/posts.js:1404]: Thank you 🤗

**Review by prathmeshjagtap — APPROVED** [review]
Code is Clean 
like the Folder structure
Functionality is Working Properly

**Review by t007rushi — APPROVED** [review]
Quite good use of ES6 syntax 
Nice naming convention followed
UI is 🔥
Good work good to go

**Pull request merged**